org.pwall.Pwall.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "org.pwall.Pwall",
```

Comment:
is `pwall.org` under the control of the developers? It doesn't look like it has anything.

Reply:
I've no domain registered for this software. The idea was to make a correct id in 3 part. For the moment the domain is available...

Reply:
Actually it's a requirement. You can use github if you don't have a domain.

https://docs.flatpak.org/en/latest/conventions.html?highlight=app-id#application-ids

Reply:
As you said, I can use github, I would think to use com.github.ltiber.Pwall 
But the documentation says it's better to use something like io.github.pwall.Pwall
but I'm not registered in github.io
any suggestion ?